cli/command/environment_command.go

```diff
@@ -1,10 +1,11 @@
 package command
 
 import (
-	"github.com/quintilesims/layer0/common/models"
-	"github.com/urfave/cli"
 	"io/ioutil"
 	"strconv"
+
```

Comment:
[nit] can you please configure your IDE to not add newlines into the import block?